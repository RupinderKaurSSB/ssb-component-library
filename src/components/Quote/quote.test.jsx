import React from 'react';
import {shallow} from 'enzyme';
import Quote from './index';

describe('Quote component', () => {
	test('Matches the snapshot', () => {
		const wrapper = shallow(<Quote>«Quote»</Quote>);
		expect(wrapper).toMatchSnapshot();
	});
	test('Toggles classNames correctly', () => {
		const wrapper = shallow(<Quote negative>«Quote»</Quote>);
		expect(wrapper.hasClass('negative')).toEqual(true);
	});

});
