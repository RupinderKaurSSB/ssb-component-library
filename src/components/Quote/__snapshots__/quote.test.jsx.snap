// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component Matches the snapshot 1`] = `
<span
  className="ssb-quote"
>
  «Quote»
</span>
`;
